LEARN.md

```diff
@@ -0,0 +1,164 @@
+
+```
+$ make new_learn_module
+...
+
+Module name (e.g., pulumi-101-aws): pulumi-101-azure
+docs/content/learn/pulumi-101-azure created
+
+...
+Topic name (e.g., basics): basics
+docs/content/learn/pulumi-101-azure/basics created
+```
+
+When you want to create a topic for a module that already exists, use the topic-specific helper:
+
+```
+$ make new_learn_topic
+```
+
+The new-topic helper will ask what module your new topic should belong to, so you'll need to give it the name of the folder that contains is -- for example, to create a new topic for the `pulumi-101-azure` module, enter `pulumi-101-azure`:
+
+```
+$ make new_learn_topic
+...
+
+Module name (e.g., pulumi-101-aws): pulumi-101-azure
+Topic name (e.g., basics): advanced-basics
+docs/content/learn/pulumi-101-azure/advanced-basics created
+```
+
+At this point, your new topic is created and you're ready to start writing. Each file should contain annotated frontmatter explaining how each property works, and if you find you don't have something you need, feel free to reach out in the #docs channel.
+
+## What's a module? What's a topic?
+
+A topic is a single focused lesson aimed at teaching someone how to do something -- ideally, something that can be completed in a relatively short period of time, like 30 minutes or less.
+
+A module is a collection of topics that share a common theme and that are organized (usually linearly) in the interest of teaching a subject that's not easily covered in a standalone tutorial, example, or blog post.
+
+### How do they differ from tutorials and examples?
+
+These definitions aren't perfect, but they should hopefully help clarify the differences.
+
+* An **example** is single Pulumi program (possibly expressed in multiple languages) with a concise README and a maybe a [Deploy with Pulumi button](https://www.pulumi.com/docs/intro/console/extensions/pulumi-button/). Examples are meant to serve as references; they're like snapshots of a finished product. Most of the programs in the [examples repo](https://github.com/pulumi/examples) today fall into this category.
+
+* A **tutorial** is a guided walkthrough that often ends with a finished example. Like a blog post (and unlike an example), it's written in a familiar tone, generally in the second person ("you/your"), and it's aimed at having the reader learn by doing. Whereas an example might say, _Here's a finished thing you can look at and use_, a tutorial says, _Let's build a thing together_, from start to finish.
+
+* A **module** is more like a chapter in a book, its topics like sections within that chapter. Stylistically, a topic may be written like a tutorial, but it may not always end with a concrete result; instead, a topic might just tell you how to do something, or walk you through how something works. By the end of a module, though, you should have a fairly thorough understanding of whatever it is the module set out to communicate at the start.
```

Comment:
```suggestion
* A **module** is more like a chapter in a book, its topics like are sections within that chapter. Stylistically, a topic may be written like a tutorial, but it may not always end with a concrete result; instead, a topic might just tell you how to do something, or walk you through how something works. By the end of a module, though, you should have a fairly thorough understanding of whatever it is the module set out to communicate at the start.
```